Package.swift

```diff
@@ -13,23 +13,29 @@ let package = Package(
             targets: ["Noora"]
         ),
         .executable(
-            name: "cli",
-            targets: ["cli"]
+            name: "examples-cli",
+            targets: ["examples-cli"]
```

Comment:
I'd still have the example in a separate `Package.swift`, I don't think the DX will suffer. But if you feel strongly about it, we can keep it as-is.

Reply:
Do you have an example of how the DX would suffer?